experiments/golden-results/UKNI-Information-Barrier-summary.txt

```diff
@@ -1,13 +1,8 @@
-Occurs: 47 times
+Occurs: 46 times
```

Comment:
I'm obviously happy to see these going down but do you see why this change reduces the occurrences of this error?